### ISSUE TITLE
I can't do statement anymore

### ISSUE THREAD

**Issue by Laurencebl1** [leifeld/dna#226]
Sometimes the software blocks me from coding. When I start my day I am able to make statements and for some unknown reason and at very random times, DNA simply stops offering me the ability to make a statement. I can still underline a sentence but when I right click my mouse nothing happens. 

My computer did not go to sleep and sometimes the time between opening the software and this bug is less than an hour. This can happen a few times in a working day. 

I can't attach a backup of an error report here since I can't save these errors. Another problem has been created for this purpose. However, when I go to the error section and look only at errors or warning, there is nothing. No errors. 

When this happens, I have to completely close the software and reopen it to make it work again. 

I am on DNA 3.0.0 , macOS Monteray, Java version is Temurin-16.0.2+7

**Comment by leifeld** [comment]
Hi there. I have a theory as to what may be happening: I think you are working on the remote server, right? It may happen that the connection to the server is interrupted or cannot be established for a short moment when you try to create the statement, either because your internet connection has a problem or because the server loses its connectivity for a moment.

To diagnose the problem more effectively, may I ask you to start DNA from the terminal using `java -jar dna-3.0.1.jar`? Please monitor the terminal screen in the background. If an unexpected error happens (like the one you described), you should see some weird exceptions popping up. Can you copy the whole exception log next time it happens and post it here? If possible along with any other hints about what happened that you haven't already covered in your post above. Once I have that log of exceptions, I can try to catch the exception when it happens and try to find a remedy. Thanks.

**Comment by Laurencebl1** [comment]
I tried to open java 3.0.1 with my terminal but it did not work. However, I was able to do it with java 3.0.0. Maybe there is something wrong? I write this first in my terminal "cd ~/Documents" and then "java -jar dna-3.0.1.jar" and the answer is "Error: Unable to access jarfile dna-3.0.1.jar". 

So for now, I open it by double clicking on the software in my folders. 

This problem happened again but I am not able to show you anything. The DNA software always indicates that there is no error.

**Comment by leifeld** [comment]
Did you put the `dna-3.0.1.jar` file in the `Documents` directory before trying it?

**Comment by Laurencebl1** [comment]
No. I just did it. Sorry, this is obviously not my strong suit. I'll let you know when it stops working again. Thank you.

**Comment by Laurencebl1** [comment]
It just happened. I opened the software a minute ago. Here is the terminal report
[Résultat de Terminal2.pdf](https://github.com/leifeld/dna/files/8043918/Resultat.de.Terminal2.pdf)

**Comment by leifeld** [comment]
Thanks, this is helpful. To figure out what is happening, I need some additional information:

1. Did you reorder the columns of the document table some time before this happened?
2. Did you deactivate any columns in the document table during the session in which this problem happened?

It's fine if you did, but I need to know to identify the problem.

**Comment by leifeld** [comment]
Also, can you perhaps take a screenshot of the main window right after this problem happens and post it here? Thanks.

**Comment by Laurencebl1** [comment]
Hello, yes i played with the columns and i thinks there's a link because it happens often after that. But not every time. I usually withdrawn (or try to withdrawn) Time, Coder, Author, Source, Section and type. Then I play with my mouse to make some smaller and some larger columns. 

I will take a screenshot the next time and post it here.
